### PR TITLE
Pass HTML error codes for abid-staging through the LB

### PR DESCRIPTION
**Pull request by acozine**
This is part of our work on #2227. It still does not work (though we had one glorious moment where we did get a 500 error passed through). For the benefit of Future Us, we memorialized our attempted changes on a branch. Obviously this will stay a draft PR until it DOES work.

### CODE REVIEW

**Comment by acozine** [comment]
Today we got this partially working. The necessary fixes/changes include:
1. add the `proxy_intercept_errors on;` directive in the `location` block of the site config (our test site config is `abid-staging`)
2. remove or comment out the inclusion of `/etc/nginx/conf.d/templates/staging-maintenance.conf` - this was active and working but uses the old syntax, and you can't have two `error_page` definitions 
3. include `/etc/nginx/conf.d/templates/errors.conf` - not sure if this should be in the `location` block or in the `server` block, right now it's in both
4. on the load balancer, copy `/var/local/www/default/index.html` (which is the deployed file created from the `roles/nginx/files/www/error.html` file) to `/var/local/www/default/error.html` (which is what the errors.conf file is looking for), with the correct permissions
5. add a specific error code you want to return, using `=XXX`, after the list of error codes to capture, in `errors.conf` (on the LB I set it to `=503`, in the git branch I've set it to `=500`)

With these changes, I consistently get the right results:
 - at the command line:
```
alicia@PULaptop ~ % curl -I https://abid-staging.princeton.edu
HTTP/2 503 
server: nginx/1.23.4
date: Mon, 31 Jul 2023 22:51:53 GMT
content-type: text/html
```
- in the browser:
Screenshot 2023-07-31 at 6 05 52 PM

Still not sure how to get the LB to pass "whatever the upstream error code is", but we're closer!

**Comment by hackartisan** [comment]
Awesome, look forward to hearing more about this. For step 5, does pass-through work if you use the plain `=` instead of, e.g., `=503`?

File: roles/nginxplus/files/conf/http/abid-staging.conf
Comment (kayiwa) [inline roles/nginxplus/files/conf/http/abid-staging.conf:1]: 🎉

**Review by kayiwa — APPROVED** [review]
OK! Thanks @acozine

**Pull request merged**